### ISSUE TITLE
Not relativizing paths with named library

### ISSUE THREAD

**Issue by ElZombieIsra** [luanpotter/vscode-dart-import#9]
The extension doesn't relativize paths when they are followed with `as`, `show` or `hide` keywords.

Ej.

```
import 'package:example/blocs/landingPage/bloc.dart';
import 'package:example/components/appBar.dart';
import 'package:flutter/material.dart';
import 'package:example/style.dart' as Style;
```

Will be relativized to this 

```
import 'package:example/style.dart' as Style;
import 'package:flutter/material.dart';

import '../../blocs/landingPage/bloc.dart';
import '../../components/appBar.dart';
```

**Comment by luanpotter** [comment]
Didn't though of that! Probably requires updating the regex to match that last bit as well inside the capture group :)

**Comment by luanpotter** [comment]
Fixed in `0.1.5`.
